OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -99,6 +103,15 @@ public bool CanTarget(Actor self, Target target, List<Actor> othersAtTarget, ref
 				if (self.World.Map.Contains(location))
 				{
 					cursor = "ability";
+
+					// Set also PrimaryBuilding with Ctrl and only this one building of the same type selected
```

Comment:
Following up on @pchote's comment about decoupling these two traits, "PrimaryBuilding" should be removed from here.
Ex:
```C#
					// Notify force-set `RallyPoint` order watchers with Ctrl and only if this is the only building of its type selected.
```